api/client/options.proto

```diff
@@ -18,6 +18,10 @@ message RequestOptions {
   google.protobuf.UInt32Value request_body_size = 3 [(validate.rules).uint32 = {lte: 4194304}];
 }
 
+message RequestOptionses {
```

Comment:
Also, maybe add a comment about the purpose of this message?
Also, `RequestOptionses` reads a bit unnatural to me. After reading some more, there seems to be a naming conflict with two things actually competing for the name RequestOptions... I wonder: can we figure out a better name? I'm not 100% sure if this is on target, but to float a suggestion something like `TrafficProfile` instead?

Reply:
Renaming it to RequestOptionsList. I hope that works better? I know calling it a list might be misleading.